cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -68,6 +68,7 @@ packageCache:
   medium: ""
   sizeLimit: 5Mi
```

Comment:
Wishing I had stuck these under an `emptyDir` entry but alas, not worth breaking folks now 😞